Formula/vgmstream.rb

```diff
@@ -25,6 +37,8 @@ class Vgmstream < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "cb21895ea0256c867a24bc300cc5bc254c2e769e4e74a4cb2766ae9f5662fe57"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```

Comment:
Let's add a comment here that these can be removed when the patches are no longer needed.

Reply:
They are needed to build `celt`, one of the dependencies of vgmstream, they can't be removed. The patches are for fixing errors caused by cmake, more details: https://github.com/vgmstream/vgmstream/issues/1069.